client/app/components/layout/SideNav.jsx

```diff
@@ -1,11 +1,47 @@
 import React from 'react';
 import { Button } from 'react-bootstrap';
+import moment from 'moment';
+import _ from 'lodash';
 
-export default () => (
-  <aside className="sidebar">
-    <Button bsStyle="primary" className="btn-block">Create Event</Button>
-    <Button bsStyle="primary" className="btn-block">Home</Button>
-    <Button bsStyle="primary" className="btn-block">Next Week</Button>
-    <Button bsStyle="primary" className="btn-block">Previous Week</Button>
-  </aside>
-);
+export default class SideNav extends React.Component {
+  static propTypes = {
+    date: React.PropTypes.string
+  };
+
+  constructor(...args) {
+    super(...args);
+
+    _.bindAll(this,
+      '_startingDate', '_nextWeek', '_previousWeek', '_dateParam');
```

Comment:
Why do you need to bind all these methods? None of them are being used as handlers